systemd-udevd/pkg.yaml

```diff
@@ -0,0 +1,114 @@
+        mkdir -p /usr/bin
+        ln -sf /toolchain/bin/env /usr/bin/env
+        ln -sf /toolchain/bin/python3 /toolchain/bin/python
+
+        pip3 install jinja2 ninja
+
+        # Patches for musl from openembedded-core, MIT licensed
+        # Also replicated by Chimera Linux: https://github.com/chimera-linux/cports/tree/001642bd7691349e95574d93e325bb05d5dcb2c1/main/udev/patches
+        patch -p1 < /pkg/patches/0004-missing_type.h-add-comparison_fn_t.patch
+        patch -p1 < /pkg/patches/0005-add-fallback-parse_printf_format-implementation.patch
+        patch -p1 < /pkg/patches/0006-don-t-fail-if-GLOB_BRACE-and-GLOB_ALTDIRFUNC-is-not-.patch
+        patch -p1 < /pkg/patches/0008-Use-uintmax_t-for-handling-rlim_t.patch
+        patch -p1 < /pkg/patches/0009-don-t-pass-AT_SYMLINK_NOFOLLOW-flag-to-faccessat.patch
+        patch -p1 < /pkg/patches/0010-Define-glibc-compatible-basename-for-non-glibc-syste.patch
+        patch -p1 < /pkg/patches/0011-Do-not-disable-buffering-when-writing-to-oom_score_a.patch
+        patch -p1 < /pkg/patches/0012-distinguish-XSI-compliant-strerror_r-from-GNU-specif.patch
+        patch -p1 < /pkg/patches/0013-avoid-redefinition-of-prctl_mm_map-structure.patch
+        patch -p1 < /pkg/patches/0014-do-not-disable-buffer-in-writing-files.patch
+        patch -p1 < /pkg/patches/0016-Handle-missing-gshadow.patch
+        patch -p1 < /pkg/patches/0017-missing_syscall.h-Define-MIPS-ABI-defines-for-musl.patch
+        patch -p1 < /pkg/patches/0019-Adjust-for-musl-headers.patch
+        patch -p1 < /pkg/patches/0022-sd-event-Make-malloc_trim-conditional-on-glibc.patch
+        patch -p1 < /pkg/patches/0023-shared-Do-not-use-malloc_info-on-musl.patch
+        patch -p1 < /pkg/patches/0024-avoid-missing-LOCK_EX-declaration.patch
+        patch -p1 < /pkg/patches/0025-include-signal.h-to-avoid-the-undeclared-error.patch
+        patch -p1 < /pkg/patches/0026-undef-stdin-for-references-using-stdin-as-a-struct-m.patch
+        patch -p1 < /pkg/patches/0027-adjust-header-inclusion-order-to-avoid-redeclaration.patch
+        patch -p1 < /pkg/patches/0028-build-path.c-avoid-boot-time-segfault-for-musl.patch
+
+        # Own patches
+        patch -p1 < /pkg/patches/0001-chore-musl-headers-compatibility-for-udev.patch
+
+        # for full build add this:
+        #-Dnss-mymachines=disabled -Dnss-myhostname=false -Dnss-resolve=disabled -Dnss-systemd=false
+        meson setup build \
+          --buildtype=release \
+          -Dmode=release \
+          -Dversion-tag="{{ .systemd_version }}-talos" \
+          -Dselinux=enabled \
+          -Dblkid=enabled \
+          -Dkmod=enabled \
+          -Dopenssl=disabled \
+          -Dgshadow=false \
+          -Dutmp=false \
+          -Duserdb=false \
+          -Ddbus=disabled \
+          -Dglib=disabled \
+          -Dlink-udev-shared=false \
+          -Dman=disabled \
+          -Dtests=false \
+          -Dwerror=false
```

Comment:
We get some untrue warnings here. A compiler update might make those go away.

![image](https://github.com/user-attachments/assets/82d77419-e400-48e9-91f8-d808e56b6520)
![image](https://github.com/user-attachments/assets/93d83f0c-ab59-4529-ae17-773622c379a9)